rocAL/rocAL/source/master_graph.cpp

```diff
@@ -1126,11 +1121,13 @@ void MasterGraph::stop_processing()
         _output_thread.join();
 }
 
-MetaDataBatch * MasterGraph::create_coco_meta_data_reader(const char *source_path, bool is_output)
+MetaDataBatch * MasterGraph::create_coco_meta_data_reader(const char *source_path, bool is_output, MetaDataReaderType reader_type, MetaDataType label_type, float sigma, unsigned pose_output_width, unsigned pose_output_height)
```

Comment:
looks like argument sigma is not used. What is the use of that argument

Reply:
It will be used in Target generation
In the current PR we have not yet added support for target generation support ,so it seems like it is not used. We will add the support the support for Target generation in the upcoming commits